examples/configs/sensors/Mic_Yeti_sens_1.yml

```diff
@@ -1,5 +1,5 @@
 sensor_id: Mic_Yeti_sens_1
-temporal_res: 44100.0
-spatial_res_x: 1024.0
+RATE: 44100.0
+CHUNK: 1024.0
```

Comment:
I would name this sample_rate and sample_chunk_size

Reply:
yeah, ok. same argument for the chosen names as above, but I can change them if you prefer.